types/mongodb/index.d.ts

```diff
@@ -1426,6 +1426,25 @@ export type UpdateQuery<TSchema> = {
     $bit?: {
         [key: string]: { [key in 'and' | 'or' | 'xor']?: number };
     };
+} | Array<UpdateAggregation<TSchema>>;
```

Comment:
this means that users will  have to check for array-or-not whenever they use UpdateQuery. Is this acceptable?